kola/tests/misc/ntp.go

```diff
@@ -62,13 +60,12 @@ func NTP(c cluster.TestCluster) error {
 			return fmt.Errorf("unexpected systemd-timesyncd status: %v", out)
 		}
 
-		plog.Info("systemd-timesyncd.service is working!")
 		return nil
 	}
 
 	err = util.Retry(60, 1*time.Second, checker)
 	if err != nil {
-		return nil
+		return err
```

Comment:
It seems cleaner to just `return util.Retry(...`, but LGTM if that violates some convention